Framework/API/inc/MantidAPI/WorkspaceFactory.h

```diff
@@ -77,8 +78,8 @@ class MANTID_API_DLL WorkspaceFactoryImpl
                               const size_t &NVectors, const size_t &XLength,
                               const size_t &YLength) const;
 
-  void initializeFromParent(const MatrixWorkspace_const_sptr parent,
-                            const MatrixWorkspace_sptr child,
+  void initializeFromParent(const MatrixWorkspace &parent,
```

Comment:
Yes. I did not bother with it since `initializeFromParent` is basically made obsolete by the new factory functions. That is, we can either change also `child` (wasting effort), or use the new factory right away. I'll think about what is better now.